x-pack/platform/plugins/shared/alerting/common/routes/rule/apis/create/schemas/v1.ts

```diff
@@ -189,6 +189,7 @@ export const createBodySchema = schema.object({
   notify_when: schema.maybe(schema.nullable(notifyWhenSchemaV1)),
   alert_delay: schema.maybe(alertDelaySchemaV1),
   flapping: schema.maybe(schema.nullable(flappingSchemaV1)),
+  dashboards: schema.maybe(schema.arrayOf(schema.object({ id: schema.string() }))),
```

Comment:
I see some keys (like flapping for example) use `schema.nullable` so I am wondering if I should change to 

```
dashboards: schema.maybe(schema.nullable(schema.arrayOf(schema.object({ id: schema.string() }))))
```